Examples/GeoShape.md

```diff
@@ -0,0 +1,123 @@
+# GeoShape Fields Usage In RediSearch
```

Comment:
Can we rename the file GeoShapeQueryExample.md? Our other examples follow a similar pattern.